docs/installation.md

```diff
@@ -19,7 +19,19 @@ pipx works on macOS, linux, and Windows.
 ```
 brew install pipx
 pipx ensurepath
-sudo pipx ensurepath --global # optional to allow pipx actions in global scope. See "Global installation" section below.
+```
+
+#### Additional (optional) commands
+
+To allow pipx actions in global scope. See "Customising your installation" section below.
+```
+sudo pipx ensurepath --global
+```
+
+To prepend the pipx bin directory to PATH instead of appending it. See "Customising your installation" section below.
+
+```
+sudo pipx ensurepath --prepend
 ```
```

Comment:
I'd rephrase this to make it more concise, and link to the actual customization documentation:


#### Additional (optional) commands

To allow pipx actions in global scope.
```
sudo pipx ensurepath --global
```

To prepend the pipx bin directory to PATH instead of appending it.

```
sudo pipx ensurepath --prepend
```

For more details, refer to [Customising your installation](installation.md#customising-your-installation).

Reply:
Makes sense, thanks! Addressed in 6fc30c07dcb7f567e9a827f071e0a92ff31c6833.